### PR TITLE
Added create note UI

### LINKED ISSUE #5: Create new note UI

### PR DESCRIPTION
**Pull request by shreyakurian02**
- Fixes #5

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 1]
```diff
@@ -26,36 +31,57 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
   return (
     <Formik
       initialValues={note}
-      validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
+      validationSchema={VALIDATION_SCHEMA}
       onSubmit={handleSubmit}
     >
       {({ isSubmitting }) => (
-        <FormikForm className="w-full">
+        <FormikForm noValidate className="w-full">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:38]: What's the need for this `noValidate` class here?

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 2]
```diff
@@ -26,36 +31,57 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
...
+              rows={1}
+            />
+            <Select
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label={t("common.assignedContact")}
+              name="assignedContact"
+              options={ASSIGNED_CONTACTS}
+              placeholder={t("placeholder.selectRole")}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:62]: The `name` is using the keyword `contact`. But the placeholder is using the keyword `role`. Let's stick onto  one of them.

**Review by yedhink — COMMENTED** [review]
@shreyakurian02

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 1 above)
Comment (shreyakurian02) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:38]: Without noValidate the browser validations are shown before the formik's validations
image

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:38]: Got it. Thanks for clarifying.

**Pull request merged**